### ISSUE TITLE
The Makefiles appear mostly broken

### ISSUE THREAD

**Issue by ld-cd** [Ravenslofty/74xx-liberty#18]
I'm attempting to build the pwmled example, but upon running make I get the following error: 
`make: *** No rule to make target 'pwmled.net'.  Stop.`

This appears to be because 74series.lib has been renamed to 74ac.lib. Upon copying 74series to 74ac some of the basic examples work (like counter), but pwmled and some of my own code fails with errors like the following:

```
aled@blackdufous-~/build/74xx-liberty/kicad: KICAD_SYMBOL_DIR=/usr/share/kicad/library/ make pwmled.net
yosys -q -s ../synth_74.ys -p "write_json pwmled.json" ../benchmarks/pwmled.v
python3 generate_netlist.py pwmled.json
Traceback (most recent call last):
  File "generate_netlist.py", line 192, in <module>
    create_chips(chip_types, nets)
  File "generate_netlist.py", line 178, in create_chips
    raise Exception("%s not handled" % typ)
Exception: \74AC86_4x1XOR2 not handled
make: *** [Makefile:9: pwmled.net] Error 1
rm pwmled.json
```

**Comment by ld-cd** [comment]
My guess is that its related to the rename in the most recent commit

**Comment by ZirconiumX** [comment]
I think this got fixed by 66e84ea?

**Comment by ld-cd** [comment]
Yup, looks like it's working!
